source/_posts/2015-03-16-bath-ruby.markdown

```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+title: "Bath Ruby Conference 2015"
+date: 2015-03-16 22:18:48 +0000
+comments: true
+author: Codebar
```

Comment:
I think personalisation is nice. Let's add the contributors names, so you and anyone else helping with the post.